marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -162,6 +162,19 @@ describe('json scenario for snippet on browse documents page', () => {
     detailPage.getDocumentJSON().should('exist');
   });
 
+  it('verify detail page source and instance tooltips', () => {
+    browsePage.search('Powers');
+    browsePage.getSourceViewIcon().trigger('mouseover');
+    cy.contains('Show the complete JSON').should('exist');
+    browsePage.getInstanceViewIcon().trigger('mouseover');
+    cy.contains('Show the processed data').should('exist');
+    browsePage.getSourceViewIcon().click();
+    detailPage.getSourceView().trigger('mouseover');
+    cy.contains('Show the complete JSON').should('exist');
+    detailPage.getInstanceView().trigger('mouseover');
+    cy.contains('Show the processed data').should('exist');
```

Comment:
Can you pls change all the 'exist' to 'be.visible'. 
'exist' condition might pass even when the tooltip is available in the DOM(which is true for some tooltip I think) but not visible.